### PR TITLE
fix `stamp = 0` in rust_binary_without_process_wrapper

### PR DESCRIPTION
**Pull request by krasimirgg**
Stamping is not supported when building without process_wrapper:
https://github.com/bazelbuild/rules_rust/blob/8df4517d370b0c543a01ba38b63e1d5a4104b035/rust/private/rustc.bzl#L955

### CODE REVIEW

**Comment by UebelAndre** [comment]
Why not remove the stamping functionality entirely from the rule? There's already some unique logic for generating the attributes:
https://github.com/bazelbuild/rules_rust/blob/0.1.0/rust/private/rust.bzl#L931-L956

**Comment by krasimirgg** [comment]
> Why not remove the stamping functionality entirely from the rule? There's already some unique logic for generating the attributes: https://github.com/bazelbuild/rules_rust/blob/0.1.0/rust/private/rust.bzl#L931-L956

Thank you! Done.

**Review by UebelAndre — APPROVED** [review]
Looks good to me, thanks!

**Comment by hlopko** [comment]
There is https://github.com/hlopko/rules_rust/blob/19aa1216358e84a0bb538008d333e6356697c49f/rust/private/rustc.bzl#L944 that covers some attributes, and this PR handles `stamp` differently. We should aim for consistency.

@UebelAndre I actually wanted @scentini to take a look and give feedback before merging. I assumed that by adding them to reviewers this was implied. Would you prefer if I commented something in the spirit "reviewers, please don't merge until we all approve" in the future?

**Comment by hlopko** [comment]
Oh I misread the code, this is actually handling the issue quite nicely, thank you!

**Comment by UebelAndre** [comment]
> @UebelAndre I actually wanted @scentini to take a look and give feedback before merging. I assumed that by adding them to reviewers this was implied. Would you prefer if I commented something in the spirit "reviewers, please don't merge until we all approve" in the future?

Sorry, next time I'll wait. Wasn't sure what the procedure was. Now I know 😅

**Pull request merged**
